benchmarks/python/core.py

```diff
@@ -146,6 +147,21 @@ def unary_bwd_torch(inputs: List):  # [output, grad_out]
     inputs[0].backward(inputs[1], retain_graph=True)
 
 
+def compute_total_iobytes(tensor_props: dict):
```

Comment:
Based on docstring this could be
```suggestion
def compute_total_iobytes(tensor_props: dict[str, tuple[tuple[int, ...], torch.dtype]]):
```